### ISSUE TITLE
navbarlinks outcome behaviour

### ISSUE THREAD

**Issue by georgemoralis** [TheCoder4eu/BootsFaces-OSP#191]
I have a very weird behaviour
I used the http://bootsfaces.net/examples/dashboard.jsf  example

if i add a outcome in a navbarlink , it seems to apply to all navbarlinks i have . So clicking to any of this it executes the outcome although it is not define in it

**Comment by stephanrauh** [comment]
Strange indeed. However, I didn't manage to reproduce the error. Can you post a code snippet? And which version of BootsFaces are you using?

**Comment by TheCoder4eu** [comment]
Hi, sorry for the delay in the answer.
I am testing the page in the test app I used to develop the Dashboard example.

I tried to modify some navlink, but everything seems to work fine.
On a rendered HTML basis everything is looking as it should be.
Original link:
```
<a tabindex="-1" role="menuitem" href="#">Settings</a>
```
Link with outcome and without href:
```
<a tabindex="-1" role="menuitem" href="/TestBsF/page/test/theme.xhtml">Profile</a>
```
Original link without href:
```
<a tabindex="-1" role="menuitem" href="/TestBsF/page/test/dashboard.xhtml">Help</a>
```
We caould have a look to a snipped of code, otherwise it could be an issue in the old release, did you try the same with v0.8.0?
